client/src/components/Modal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+const backdropStyle = {
+    position: 'fixed',
+    top: 0,
+    bottom: 0,
+    left: 0, 
+    right: 0,
+    backgroundColor: 'rgba(0,0,0,0.3)',
+    padding: 50
+  }
+
+const modalStyle = {
+    backgroundColor: '#fff',
+    borderRadius: 5,
+    maxWidth: 500,
+    minHeight: 300,
+    margin: '0 auto',
+    padding: 30,
+    position: "relative"
+}
+
+const footerStyle = {
+    position: 'absolute',
+    bottom: 20
+}
+
+class Modal extends React.Component {
+
+    onClose(e) {
+        this.props.onClose && this.props.onClose(e);
```

Comment:
If you're always passing down an onClose prop here you don't necessarily need to do a conditional check for it's existance here but it never hurts!